developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -806,6 +806,14 @@ Operand Stack:
 
 In this example, we'll use the `HelloBlockchain` in [move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples).
 
+:::important
+As an open source project, the source code as well as compiled code published to the Aptos blockchain is inherently open by default. This means code you upload may be downloaded and even profited by from on-chain data. Even without source access, it is possible to regenerate Move source from Move bytecode. To make this harder and disable source access, publish with the `--included-artifacts none` argument, like so:
```

Comment:
"even profited by from on-chain data" is speculative IMO. I'd prefer removing that part.

Reply:
Same for "To make this harder".

Reply:
> "even profited by from on-chain data" is speculative IMO. I'd prefer removing that part.

Actually, this was the case raised in the internal #Community channel that prompted this update.

Reply:
+1 with Kevin. It will be open-source sooner or later. This is just an option so we should not be biased here.

Reply:
> +1 with Kevin. It will be open-source sooner or later. This is just an option so we should not be biased here.

Done!